.github/DangerFiles/TestOrchestrator.rb

```diff
@@ -1,3 +1,11 @@
+#!/usr/bin/env ruby
+
+# Warn when there is a big PR
+warn("Big PR, try to keep changes smaller if you can.", sticky: true) if git.lines_of_code > 1000
+
+# Redirect contributors to PR to dev.
+fail("Please re-submit this PR to the dev branch, we may have already fixed your issue.", sticky: true) if github.branch_for_base != "dev"
```

Comment:
Does the whole check fail? Wondering about when we do PRs to master for cherry picks

Reply:
I believe it only fails the Danger status check on the PR.  It should not stop the execution of tests or do much of anything else negative.

However, while writing this comment I realized it is actually completely pointless with the current configuration since CI is only triggered on `dev`.  So I think we have two choices:

1. Add `master` as a branch to trigger PRs.  We probably want to keep the line then and ignore the fail.  (Although I thought we typically just merge dev to master without a PR?).
2. Remove the warning since CI doesn't run against `master`.  

I think I like option #1 more?

Reply:
sounds good! it's been a while but this was the scenario I was thinking of https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/3686

Reply:
I double checked this on the Android testing setup.  It just displays this error, which is fine in the rare event we are cherry picking to master.

<img width="943" alt="Screenshot 2025-01-29 at 5 27 23 PM" src="https://github.com/user-attachments/assets/f872b5de-5bdd-4392-ad29-98acd1207a16" />